### PR TITLE
m4b-tool: add page

### PR DESCRIPTION
**Pull request by C4N-6**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): v.0.4.2**

### CODE REVIEW

File: pages/common/m4b-tool.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# m4b-tool
+
+> A tool merge, split or and manipulate audiobook files with chapters.
+> More information: <https://github.com/sandreas/m4b-tool>.
+
+- Create an audiobook with the audio files in inputted directory:
```
Comment (gutjuri) [inline pages/common/m4b-tool.md:6]:
```suggestion
- Create an audiobook with the audio files in the input directory:
```

File: pages/common/m4b-tool.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# m4b-tool
+
+> A tool merge, split or and manipulate audiobook files with chapters.
```
Comment (gutjuri) [inline pages/common/m4b-tool.md:3]:
```suggestion
> Merge, split and manipulate audiobook files with chapters.
```

File: pages/common/m4b-tool.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://github.com/sandreas/m4b-tool>.
+
+- Create an audiobook with the audio files in inputted directory:
+
+`m4b-tool merge "data/my-audio-book/" --output-file="data/merged.m4b"`
```
Comment (gutjuri) [inline pages/common/m4b-tool.md:8]:
```suggestion
`m4b-tool merge {{path/to/input_directory/}} --output-file={{path/to/merged.m4b}}`
```

File: pages/common/m4b-tool.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+- You can make chapters use file names by adding the flag --use-filenames-as-chapters:
+
+`m4b-tool merge "data/my-audio-book/" --output-file="data/merged.m4b" --use-filenames-as-chapters`
```
Comment (gutjuri) [inline pages/common/m4b-tool.md:12]:
```suggestion
- Make chapters using the input files' names:

`m4b-tool merge {{path/to/input_directory/}} --output-file={{path/to/merged.m4b}} --use-filenames-as-chapters`
```

**Review by gutjuri — COMMENTED** [review]
Hi, thank you very much for you contribution! I have some suggestions. In particular:

- Use {{ .. }} for placeholders
- Use the imperative form for descriptions

I will approve of this PR when these changes are applied :)

**Comment by C4N-6** [comment]
Thank you for the suggestions. I have committed the changes you have requested.

File: pages/common/m4b-tool.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+> Merge, split and manipulate audiobook files with chapters.
+> More information: <https://github.com/sandreas/m4b-tool>.
+
+- Create an audiobook with the audio files in the input directory:
+
+`m4b-tool merge {{path/to/input_directory/}} --output-file={{path/to/merged.m4b}}`
+
+- Make chapters using the input files' names:
+
+`m4b-tool merge {{path/to/input_directory/}} --output-file={{path/to/merged.m4b}} --use-filenames-as-chapters`
```
Comment (acuteenvy) [inline pages/common/m4b-tool.md:12]:
```suggestion
`m4b-tool merge {{path/to/input_directory}} --output-file={{path/to/merged.m4b}}`

- Make chapters using the input files' names:

`m4b-tool merge {{path/to/input_directory}} --output-file={{path/to/merged.m4b}} --use-filenames-as-chapters`
```

**Review by gutjuri — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

File: pages/common/m4b-tool.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# m4b-tool
+
+> Merge, split and manipulate audiobook files with chapters.
```
Comment (spageektti) [inline pages/common/m4b-tool.md:3]:
```suggestion
> Merge, split, and manipulate audiobook files with chapters.
```

**Pull request merged**